scripts/use-react-dist-tag.js

```diff
@@ -51,14 +51,7 @@ async function main(distTag) {
   // add newline for clean diff
   fs.writeFileSync(packageJsonPath, `${JSON.stringify(packageJson, null, 2)}${os.EOL}`);
 
-  try {
-    await exec(`git apply ${path.resolve(__dirname, `./react-${distTag}.diff`)}`);
-  } catch (error) {
-    // ignore if the patch doesn't exist
-    if (error.code !== 128) {
```

Comment:
This code is not necessarily tied to non-existing patches. It hid the error message that the patch was corrupted. 

I don't remember why I added it. In the end we could always leave the patch empty on certain branches.